Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,18 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name1 = "Tom Jefferson";
+            int age1 = 19;
+            int weight1 = 50;
+            float height1 = 156.5f;
+            float bmi1 = weight1 / height1 / height1 * 10000;
+            Console.WriteLine($"My name is {name1} and I am {age1} years old, my weight is {weight1} kg, I'm {height1} meters tall and my BMI is {bmi1}");
+            string name2 = "Tom Holland";
```

Comment:
This code is hard to read, it looks like a blob of code. You can play a bit smarter and use blank lines separating the two people, input gathering and bmi calculation. Also comments would be nice.

Lastly, please don't hardcode the input, but rather use 
``int age1 = int.Parse(Console.ReadLine());``
for getting input (ReadLine) and converting it to the type you need (int.Parse).